media-libs/opusfile/opusfile-0.11.ebuild

```diff
@@ -16,14 +16,16 @@ RDEPEND="media-libs/libogg
 	media-libs/opus
 	http? (
 		!libressl? ( dev-libs/openssl:0= )
-		libressl? ( dev-libs/libressl:= )
+		libressl? ( dev-libs/libressl:0= )
 	)"
 
 DEPEND="${RDEPEND}
 	doc? ( app-doc/doxygen )"
 
 REQUIRED_USE="^^ ( fixed-point float )"
 
+PATCHES=( "${FILESDIR}/http-fix-compilation-with-LibreSSL.patch" )
```

Comment:
Could you include the package and version number in patch filename? You can also make it shorter then, e.g. `opusfile-0.11-libressl.patch`. This will make it clear it only needs to apply to this version.

Reply:
Sure. Done. Thanks for the comment.